src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -122,6 +122,33 @@ const PIAFormPage = () => {
     }));
   };
 
+  /**
+   * Reset pia.review when pia.status is changed to INCOMPLETE or EDIT_IN_PROGRESS
+   */
+  const resetPiaReview = () => {
+    piaStateChangeHandler(
+      {
+        programArea: {
+          selectedRoles: [],
+          reviews: {},
+        },
+        mpo: null,
+      },
+      'review',
+      true,
+    );
+  };
+
+  useEffect(() => {
+    if (
+      pia.status === PiaStatuses.INCOMPLETE ||
+      pia.status === PiaStatuses.EDIT_IN_PROGRESS
+    ) {
```

Comment:
we may need to add an accesscontrol to this page. If they are in incomplete or editinprogress they cannot view this page

may be add this logic to this function   [const accessControl = useCallback(() => ](https://github.com/bcgov/cirmo-dpia/blob/3ffcd1dcab5d13f708db7d5839e4e524d698e43c/src/frontend/src/pages/PIAForm/index.tsx#L249C1-L249C1)

to include if they are in review page and in this mode they need to be navifated to the intake page?